cmd/describe.go

```diff
@@ -0,0 +1,55 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/redhat-developer/odo/pkg/application"
+	"github.com/redhat-developer/odo/pkg/component"
+	"github.com/redhat-developer/odo/pkg/project"
+	"github.com/spf13/cobra"
+)
+
+var describeCmd = &cobra.Command{
```

Comment:
`Args` is missing. See one of the previous commands from @containscafeine

Reply:
added `Args`